Documentation.docc/V4_API_Migration_guide.md

```diff
@@ -0,0 +1,193 @@
+but a number of updates make our Swift API more idomatic. We'll be updating this list as we continue to release betas.
+
+To start us off, Our framework name changed from `Purchases` to `RevenueCat` 😻! We also updated all references of `Purchaser` to `Customer` to be more consistent across our platform. 
+
+If you're using `Carthage`, make sure to use the new `RevenueCat.framework` or `RevenueCat.xcframework` instead of the old `Purchases`.
+
+### Xcode version requirements and updated deployment targets
+`purchases-ios` v4 requires using Xcode 13.2 or newer. 
+It also updates the minimum deployment targets for iOS, macOS and tvOS. 
+
+##### Minimum deployment targets
+|  | v3 | v4 |
+| :-: | :-: | :-: |
+| iOS | 9.0 | 11.0 |
+| tvOS | 9.0 | 11.0 |
+| macOS | 10.12 | 10.13 |
+| watchOS | 6.2 | 6.2 (unchanged) |
+
+### Known Issues
+
+#### ObjC + SPM
+If you expose any Purchases objects from one target to another (see [example project](https://github.com/taquitos/SPMBug)
+for what this could look like), that second target will not build due to a missing autogenerated header.
+Currently there is a known bug in SPM whereby Xcode either doesn't pass the RevenueCat ObjC generated interface to SPM,
+or SPM just doesn't integrate it. You can follow along: [SR-15154](https://bugs.swift.org/browse/SR-15154). 
+
+##### Workaround: 
+You must manually add the autogenerated file we committed to the repository
+[RevenueCat-Swift.h](https://github.com/RevenueCat/purchases-ios/blob/main/IntegrationTests/CommonFiles/RevenueCat-Swift.h)
+to your project, and `#import RevenueCat-Swift.h` in your bridging header. You can see how we do this in our
+[SPMIntegration project](https://github.com/RevenueCat/purchases-ios/tree/main/IntegrationTests/SPMIntegration).
+
+### StoreKit 2
+
+You can opt in to using `StoreKit 2` features when you configure `Purchases`:
+```swift
+Purchases.configure(
+	withAPIKey: "your_api_key",
+	appUserID: nil,
+	observerMode: false,
+	userDefaults: nil,
+	useStoreKit2IfAvailable: true
+)
+```
+
+### New Types
+
+To better support `StoreKit 2`, `RevenueCat v4` introduces several new types to encapsulate data from `StoreKit 1` and `StoreKit 2`:
+
+- `StoreProduct` / `RCStoreProduct`: wraps a `StoreKit.SKProduct` or `StoreKit.Product`
+- `StoreTransaction` / `RCStoreTransaction`: wraps a `StoreKit.SKPaymentTransaction` or `StoreKit.Transaction`
+- `StoreProductDiscount` / `RCStoreProductDiscount`: wraps a `StoreKit.SKProductDiscount` or `StoreKit.Product.SubscriptionOffer`
+
+### ObjC type changes
+
+`@import Purchases` is now `@import RevenueCat`
+
+| v3 | v4 |
+| ------------ | ------------------------------------- | 
+| RCPurchaserInfo | RCCustomerInfo |
+| RCTransaction | RCStoreTransaction |
+| RCPackage.product | RCPackage.storeProduct |
+| (RCPurchasesErrorCode).RCOperationAlreadyInProgressError | RCOperationAlreadyInProgressForProductError |
```

Comment:
This is the only change in this file - updating the table format from HTML format to markdown, which Docc supports

Reply:
it should look the same, though

Reply:
It also so much easier to maintain. Thanks for taking the time to do this!